### DISCUSSION TITLE
Token argument on livechat_register_visitor class

### DISCUSSION THREAD

**Discussion by mazinhomedeiros** [jadolg/rocketchat_API#195] [Q&A]
Hello,

How to deal with token argument present on livechat_register_visitor class?

I'm receiving this error: 
TypeError: 'str' object does not support item assignment

Thanks in advance.

**Comment by jadolg** [comment]
This is how this method is tested 
https://github.com/jadolg/rocketchat_API/blob/9f83de2851e381a82a72321e72030fd588fc8f36/tests/test_livechat.py#L57-L61
As you can see, a simple string (a uuid) is passed there as token.
What are you trying to use it with?
